### ISSUE TITLE
Command 'cdwr' not found

### ISSUE THREAD

**Issue by malemu17** [codefellows/code-201-prework#87]
![dotprofile](https://user-images.githubusercontent.com/46510682/64673243-76e1d900-d422-11e9-88f7-6a01e3130e28.JPG)

here attached with my updated .profile with  the provided code . But I couldn't able to open windows root  from Ubuntu it gave me the following error message . 
Command 'cdwr' not found, did you mean:

  command 'cdpr' from deb cdpr
  command 'cdw' from deb cdw
  command 'cdir' from deb cdtool

any comment please 

thanks 
Meron

**Comment by malemu17** [comment]
yes I did close and re open it  but still  I am having  the same issue

**Comment by ncarignan** [comment]
pinged student and issue was resolved before it could be documented
